examples/aws/main.tf

```diff
@@ -44,26 +43,46 @@ resource "gardener_shoot" "test_cluster" {
 
     provider {
       type = "aws"
+      infrastructure_config {
+        aws {
+          enableecraccess = true
+          networks {
+            vpc {
+              cidr = "10.50.0.0/16"
+            }
+            zones {
+              name     = "eu-central-1a"
+              internal = "10.50.112.0/22"
+              public   = "10.50.96.0/22"
+              workers  = "10.50.0.0/19"
+            }
+          }
+        }
+      }
       worker {
         name            = "cpu-worker"
         max_surge       = 1
         max_unavailable = 0
         maximum         = 2
         minimum         = 2
         volume {
-          size = "50Gi"
-          type = "Standard_LRS"
+          type = "gp2"
+          size = "30Gi"
         }
         machine {
-          type    = "m5.large"
+          type = "t3.medium"
+          image {
+            name    = "coreos"
+            version = "ami-d0dcef3"
+          }
         }
-      }
 
+        zones = ["us-east-1a", "us-east-1b", "us-east-1c"]
+      }
     }
 
     kubernetes {
       version = "1.15.4"

```

Comment:
This version of k8s is not supported by AWS any more `version = "1.17.3"`

Reply:
updated !